src/Sylius/Bundle/UserBundle/EventListener/UserDeleteListener.php

```diff
@@ -25,17 +25,17 @@
  * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
  * @author Michał Marcinkowski <michal.marcinkowski@lakion.com>
  */
-class UserDeleteListener
+final class UserDeleteListener
```

Comment:
It's worth to mention it in `UPGRADE.md` file.

Reply:
Done.